packages/nameguard-react/src/components/SettingsModal/SettingsModal.tsx

```diff
@@ -73,12 +73,10 @@ export const SettingsModal = () => {
                     <div className="space-y-4">
                       <div className="relative flex items-start">
                         <div className="flex h-6 items-center">
-                          <input
+                          <Checkbox
```

Comment:
A few UX suggestions here:

![CleanShot 2024-10-30 at 23 13 28](https://github.com/user-attachments/assets/018562c9-fd27-4e28-8f9c-46392c3d9852)

In the screenshot above, let's please think about 3 regions related to each checkbox:
1. The checkbox
2. The space between the checkbox and the label
3. The label

Currently, the logic is working like this:

1. Checkbox:
   - Hover: show hover state on checkbox
   - Hover cursor: Show "clicky hand"
   - Click: toggle checkbox selected state

2. The space between the checkbox and the label:
   - Hover: Nothing
   - Hover cursor: Nothing
   - Click: Nothing

3. The label
   - Hover: show hover state on checkbox
   - Hover cursor: Nothing
   - Click: toggle checkbox selected state

Please note how all 3 of these are working differently currently.

I believe all 3 should work exactly the same. All 3 should follow these rules:
- Hover: show hover state on checkbox
- Hover cursor: Show "clicky hand"
- Click: toggle checkbox selected state

Reply:
@lightwalker-eth Nice! done